tsl/src/fdw/scan_exec.c

```diff
@@ -476,6 +479,7 @@ fdw_scan_explain(ScanState *ss, List *fdw_private, ExplainState *es, TsFdwScanSt
 
 		if (ts_guc_enable_remote_explain)
 		{
+			Assert(fsstate);
```

Comment:
```suggestion
		if (ts_guc_enable_remote_explain && fsstate)
		{
```
My suggestion was to move the assert into the if not drop it completely

Reply:
I got that. My point is/was that if `ts_guc_enable_remote_explain` is true then `fsstate` HAS to be non-NULL always

Reply:
I'd rather get an assert or a crash which we can fix later in such a case @svenklemm

Reply:
I  disagree the code has a lot of crash bugs already so not introducing new possible crashes should be a priority. Also a crash affects all running sessions and is very disruptive so handling errors as graceful as possible seems preferable.

Reply:
I am not too convinced.. but making the change as above for now..